addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -933,4 +999,8 @@ private void updateCubeState() {
         updateState(new ChannelUID(getThing().getUID(), CHANNEL_FREE_MEMORY), new DecimalType(freeMemorySlots));
         updateState(new ChannelUID(getThing().getUID(), CHANNEL_DUTY_CYCLE), new DecimalType(dutyCycle));
     }
+
+    private boolean hasExcessDutyCycle() {
```

Comment:
This probably needs to become a public one.
The MaxDevicesHandler should not initiate the actuals refreshing if this condition exists.